go-controller/pkg/kubevirt/pod.go

```diff
@@ -57,3 +57,24 @@ func FindNetworkInfo(client *factory.WatchFactory, pod *corev1.Pod) (NetworkInfo
 	}
 	return networkInfo, nil
 }
+
+// PodIsLiveMigrationLeftOver return true if there are other pods related to
+// to it and any of them has newer creation timestamp.
+func PodIsLiveMigrationLeftOver(client *factory.WatchFactory, pod *corev1.Pod) (bool, error) {
+	vmPods, err := FindVMRelatedPods(client, pod)
+	if err != nil {
+		return false, fmt.Errorf("failed finding related pods for pod %s/%s when checking live migration left overs: %v", pod.Namespace, pod.Name, err)
+	}
+
+	for _, vmPod := range vmPods {
+		if vmPod.CreationTimestamp.After(pod.CreationTimestamp.Time) {
+			return true, nil
+		}
+	}
+
+	return false, nil
+}
+
+func PodMatchesExternalIDs(pod *corev1.Pod, externalIDs map[string]string) bool {
+	return len(externalIDs) > 1 && externalIDs[NamespaceExternalIDKey] == pod.Namespace && externalIDs[kvv1.VirtualMachineNameLabel] == pod.Labels[kvv1.VirtualMachineNameLabel]
```

Comment:
Could you put each condition in a separate line ? I think that's more readable.
```suggestion
	return len(externalIDs) > 1 && 
	    externalIDs[NamespaceExternalIDKey] == pod.Namespace && 
	    externalIDs[kvv1.VirtualMachineNameLabel] == pod.Labels[kvv1.VirtualMachineNameLabel]
```

Reply:
Have refactor this into 
```golang
/ ExternalIDContainsVM return true if the nbdb ExternalIDs has namespace                
// and name entries matching the VM                                             
func ExternalIDsContainsVM(externalIDs map[string]string, vm *ktypes.NamespacedName) bool {
    if vm == nil {                                                                                                    
        return false                                                            
    }                                                                                               
    return externalIDs[NamespaceExternalIDKey] == vm.Namespace && externalIDs[kvv1.VirtualMachineNameLabel] == vm.Name
}                                                                               
                                                                                
// ExtractVMFromPod retunes namespace and name of vm backed up but the pod      
// for regular pods return nil                                                  
func ExtractVMFromPod(pod *corev1.Pod) *ktypes.NamespacedName {                 
    vmName, ok := pod.Labels[kvv1.VirtualMachineNameLabel]                         
    if !ok {                                                                    
        return nil                                                              
    }                                                                           
    return &ktypes.NamespacedName{Namespace: pod.Namespace, Name: vmName}       
}                    
```